### PR TITLE
154 op support atengludefault

### PR DESCRIPTION
**Pull request by aarushjain29**

### CODE REVIEW

File: py/torch_migraphx/fx/converters/aten_ops_converters.py [hunk 1]
```diff
@@ -435,6 +435,14 @@ def aten_ops_sigmoid(mgx_module, node, args, kwargs):
     return acc_ops_converters.acc_ops_sigmoid(mgx_module, node, (), acc_kwargs)
 
 
+@migraphx_converter(torch.ops.aten.glu.default)
```
Comment (shivadbhavsar) [inline py/torch_migraphx/fx/converters/aten_ops_converters.py:438]: missing test for this under test/dynamo/

File: py/torch_migraphx/fx/converters/aten_ops_converters.py [hunk 2]
```diff
@@ -435,6 +435,14 @@ def aten_ops_sigmoid(mgx_module, node, args, kwargs):
...
+def aten_ops_glu(mgx_module, node, args, kwargs):
+    assert len(args) == 2
```
Comment (shivadbhavsar) [inline py/torch_migraphx/fx/converters/aten_ops_converters.py:440]:
```suggestion
    assert len(args) >= 1
```

File: tests/fx/converters/test_activations_fx.py [hunk 3]
```diff
@@ -91,6 +91,20 @@ def test_softmax(inp_size, dim):
...
+                                           ((10 ,16, 32, 64), 2)])
+def test_glu(inp_size, dim):
+    
+    inp = torch.randn(inp_size)
+    mod = FuncModule(torch.nn.functional.glu, dim=dim)
+
+    mgx_mod = convert_to_mgx(mod, [inp])
+    verify_outputs(mod, mgx_mod, inp)
+
+
```
Comment (shivadbhavsar) [inline tests/fx/converters/test_activations_fx.py:106]: add test case where dim is not provided

Comment (shivadbhavsar) [reply tests/fx/converters/test_activations_fx.py:53]: This is not resolved. This means your funcmodule should be defined as follows:
`FuncModule(torch.nn.functional.glu)`

You can do this is a seperate function or add a line where dim is None in your parametrize and wrap the above call in if dim == None clause

**Review by bpickrel — APPROVED** [review]
Nothing to add.

**Review by shivadbhavsar — CHANGES_REQUESTED** [review]
unresolved comment - just marking as requested changes so it doesnt get missed

**Review by shivadbhavsar — APPROVED** [review]
